docs: added logo (#256)
animo/siera-cli: README.md

    
</p>

> Are you looking for a graphical application? Try out or new [Siera Desktop](https://siera.animo.id/desktop)

Reviewer: Graphical application sounds a bit ... old maybe? 

Maybe something like:

```suggestion
Looking for more UI and less CLI? Try out the new [Siera Desktop](https://siera.animo.id/desktop) application.
```

> Looking for more UI and less CLI? Try out the new [Siera Desktop](https://siera.animo.id/desktop) application.

Reviewer 2: nice!
